test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/acquisitiondetails/AcquisitionDetailsControllerSpec.scala

```diff
@@ -102,6 +102,8 @@ class AcquisitionDetailsControllerSpec
   def assetTypeMessageKey(assetType: AssetType): String =
     if (assetType === IndirectDisposal) ".indirect" else ""
 
+  def expectedSubmitText(isAmend: Boolean) = if(isAmend) "Continue" else "Save and continue"
```

Comment:
these strings should be derived from `messageFromMessageKey` using the message keys